Won't compile if all .elm files are in main src/ dir

Heya, saw this on elm-discuss. I think this project is a great idea! I replaced my simple Makefile with yours, and tried it out. First issue I ran into was that it requires you two have at least one directory of .elm files inside your /src dir. The project I tried it out in is very simple, and all .elm files are directly in the "root" /src dir. Here's some output demonstrating the issue:

```
$ make
test -s src/Main.elm || echo "$main_elm" > src/Main.elm
make: *** No rule to make target 'src/**/*.elm', needed by 'build/main.js'.  Stop.
$ mkdir src/asdf
$ touch src/asdf/asdf.elm
$ make
test -s src/Main.elm || echo "$main_elm" > src/Main.elm
elm make src/Main.elm --yes --warn --output build/main.js
Success! Compiled 1 modules.                                        
Successfully generated build/main.js
...
```

This would be a PR, but I don't really know make well enough!

--- a/elm.mk
+++ b/elm.mk
@@ -1,5 +1,6 @@
 .PHONY: install server watch clean test help
 ELM_ENTRY = src/Main.elm
+ELM_FILES = $(shell find src -type f -name '*.elm')
 NODE_BIN_DIRECTORY = node_modules/.bin
 DEVD_VERSION = 0.3
 WELLINGTON_VERSION = 1.0.2
@@ -95,7 +96,7 @@ node_modules/.bin/elm-test:
 build/main.css: styles/*.scss
 	bin/wt compile -b build/ styles/main.scss
 
-build/main.js: src/*.elm src/**/*.elm
+build/main.js: $(ELM_FILES)
 	elm make $(ELM_ENTRY) --yes --warn --output $@
 
 build/interop.js: src/interop.js

